supervision/detection/tools/inference_slicer.py

```diff
@@ -50,8 +51,10 @@ class InferenceSlicer:
             `(width, height)`.
         overlap_ratio_wh (Tuple[float, float]): Overlap ratio between consecutive
             slices in the format `(width_ratio, height_ratio)`.
-        iou_threshold (Optional[float]): Intersection over Union (IoU) threshold
-            used for non-max suppression.
+        overlap_filter (OverlapFilter): Strategy for
+            filtering or merging overlapping detections in slices.
+        iou_threshold (float): Intersection over Union (IoU) threshold
+            used when filtering by overlap.
```

Comment:
Note: this inserts overlap filter between former arguments. It makes more sense stylistically, but may break implementations where keywords were not explicitly specificed.

Do we want safety or tidyness here?

Reply:
Keep it as it is.